force-app/main/default/classes/tasks/BasicTasks.cls

```diff
@@ -3,8 +3,9 @@ public class BasicTasks {
   public static Long sumIntegers(Integer a, Integer b) {
     // ignore public static.. Focus on variables
     //write your code below this line
-    Integer sum = 0;
-
+    Long a1 = a;
```

Comment:
👍  (done)

*Do you know a way to avoid creating additional variables?

Reply:
Re-writen without adding some new variables